docs/DeveloperGuide.adoc

```diff
@@ -362,9 +362,10 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |Student | Delete tasks on study plan | Can delete study tasks if I don't need it anymore
 
-|`* * *` |Student | Auto-generate event from a task | If I feel like I want to do a study task today, I can generate an event on today's schedule, or any day.
+|`* * *` |Student | Generate event from a task | If I feel like I want to do a study task today, I can generate an
+event on today's schedule so that I can keep up with my study plan.
```

Comment:
The last column is still not resolved. Maybe something like "Can set a time to complete the study task."?

Reply:
"can keep up with my study plan"